docs/framework/resiliency/chaos-engineering.md

```diff
@@ -148,8 +148,12 @@ Faults that can be leveraged to inject chaos.
 |  | Rack |  |
 |  | UPS  |  |
 |  | Datacenter |  |
+| **Functions** | Latency |  |
```

Comment:
@gunnargrosch can you move the functions to be right after Platform and before Application specific in the table?

Reply:
@absheik Moved it and commited.